lib/amazing_print/compatibility_defaults.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+unless defined? AwesomePrint
+  class AwesomePrint
+    def self.defaults=(default_values)
+      AmazingPrint.defaults = default_values
+    end
+  end
+end
```

Comment:
I'm not really a fan of defining the old `AwesomePrint` here.

Reply:
Yes, absolutely would be better to have own config file, which does not conflict